h/presenters.py

```diff
@@ -68,6 +68,29 @@ def uri(self):
         return jinja2.escape(self.annotation.target_uri)
 
     @property
+    def text(self):
+        """The body text of this annotation.
+
+        """
+
+        return  jinja2.escape(self.annotation.text)
+
+    @property
+    def quote(self):
+        """The target text of this annotation.
```

Comment:
See how other single-line API doc comments in the file are formatted, just `"""One line description."""` with no extra newlines